cms/djangoapps/contentstore/views/item.py

```diff
@@ -784,7 +784,15 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
     #instead of adding a new feature directly into xblock-info, we should add them into override_type.
     override_type = {}
     if getattr(xblock, "is_entrance_exam", None):
-        override_type['is_entrance_exam'] = xblock.is_entrance_exam
+        override_type['is_entrance_exam_section'] = xblock.is_entrance_exam
```

Comment:
I just wanted to confirm that we haven't changed this field name in any of the data models that get saved to MongoDB, because that would be a breaking change.